operatorcourier/validate.py

```diff
@@ -75,6 +76,13 @@ def _bundle_validation(self, bundle):
         if self.ui_validate_io:
             validationDict[self.csvKey] &= self._type_validation(
                 bundleData, self.csvKey, self._ui_validation_io, True)
+        if validationDict[self.pkgsKey] and repository is not None:
```

Comment:
So this check will only run for the `push` command, correct?

Reply:
That's correct. Because the repository argument is only required for push command.